multi_step_wizard/views/multi_step_wizard_views.xml

```diff
@@ -1,23 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-  <record id="multi_step_wizard_form" model="ir.ui.view">
-    <field name="name">multi.step.wizard.form</field>
-    <field name="model">multi.step.wizard.mixin</field>
-    <field name="arch" type="xml">
-      <form>
-        <field name="state" invisible="1"/>
-        <footer>
-          <div name="states_buttons" attrs="{'invisible': [('state', '=', 'final')]}">
-            <button name="open_next" string="Next" type="object" class="btn-primary"/>
-            <button string="Cancel" class="btn btn-default" special="cancel" />
-          </div>
-          <div name="final_buttons" attrs="{'invisible': [('state', '!=', 'final')]}">
-            <button string="Close" class="btn btn-primary" special="cancel" />
-          </div>
-        </footer>
-      </form>
-    </field>
-  </record>
+    <record id="multi_step_wizard_form" model="ir.ui.view">
+        <field name="name">multi.step.wizard.form</field>
+        <field name="model">multi.step.wizard.mixin</field>
+        <field name="arch" type="xml">
+            <form>
+                <field name="state" invisible="1"/>
+                <footer>
+                    <div name="states_buttons">
+                        <button name="open_next" string="Next" type="object"
+                                class="btn-primary" attrs="{'invisible': [('state', '=', 'final')]}"/>
+                        <button name="open_previous" string="Go Back" type="object"
```

Comment:
I'm not sure if all wizards support going back, so maybe this button should be disabled by default